server/src/main/java/org/elasticsearch/cli/KeyStoreAwareCommand.java

```diff
@@ -75,7 +75,7 @@ protected static void decryptKeyStore(KeyStoreWrapper keyStore, Terminal termina
         try (SecureString keystorePassword = keyStore.hasPassword() ?
             readPassword(terminal, false) : new SecureString(new char[0])) {
             keyStore.decrypt(keystorePassword.getChars());
-        } catch (SecurityException e) {
+        } catch (Exception e) {
```

Comment:
Because GeneralSecurityException can be thrown now too

Reply:
Yeah I missed that when I changed it in `BaseKeyStoreCommand`. readPassword throws a UserException and we don't want to catch it here either.

> Also KeyStoreWrapper#decrypt is declared to throw GeneralSecurityException as well. It wasn't a problem before or maybe it was just an oversight?

I think it was an oversight but it wasn't such an issue because the CLI would just exit throwing the exception. It wasn't as user friendly as a UserException with an error message but it wasn't too problematic to be noticed or reported, I'd guess